src/glossary.md

```diff
@@ -59,12 +59,27 @@ and functions that evaluate to a single value, with or without side-effects.
 
 For example, `2 + (3 * 4)` is an expression that returns the value 14.
 
+### Inherent Implementation
+
+An implementation that associates items directly to a type. These associated
+items are also called inherent: *inherent method*, *inherent function*,
+*inherent constant*.
+
+[More][inherent implementations].
+
+Contrast with [*trait implementation*][#trait-implementation].
+
 ### Initialized
 
 A variable is initialized if it has been assigned a value and hasn't since been
 moved from. All other memory locations are assumed to be initialized. Only
 unsafe Rust can create such a memory without initializing it.
 
+### Method
+
+An associated function that takes `Self`, `&Self`, `&mut Self` or `Box<Self>` as
```

Comment:
Methods are defined by the first parameter being called `self`, not just the type of the first parameter.